src/services/ChatterService.ts

```diff
@@ -25,7 +35,14 @@ class ChatterService {
 
     const { viewers } = response.data.chatters;
 
-    return viewers;
+    return this.removeBots(viewers);
+  }
+
+  private async removeBots(viewers: Participants): Promise<Participants> {
```

Comment:
I think we don't need to fetch the bots every time we fetch the current participants on `getViewers()`. Maybe we can load the bot list once at starting the service?

Reply:
This is a good point, I don't thinked it.

But it is ease to fix ;D